lib/Doctrine/DBAL/Schema/Visitor/DropSchemaSqlCollector.php

```diff
@@ -78,15 +78,18 @@ public function getQueries()
     {
         $sql = [];
 
+        /** @var ForeignKeyConstraint $fkConstraint */
```

Comment:
Would it make sense to document these on the iterators above?

Reply:
See the description.